components/brave_rewards/resources/ui/components/tableContribute/index.tsx

```diff
@@ -137,12 +137,12 @@ export default class TableContribute extends React.PureComponent<Props, {}> {
 
       if (this.props.showRemove) {
         const actionTooltip = isExcluded
-          ? getLocale('restoreSite')
+          ? getLocale('removeFromExcluded')
           : getLocale('excludeSite')
 
         cell.content.push({
           content: (
-            <Tooltip content={actionTooltip}>
+            <Tooltip rightEdge={true} content={actionTooltip}>
```

Comment:
What does `rightEdge` do?

Reply:
It moves the tooltip to be to the left of the button (instead of being centered on the button). If I leave it centered the tooltip doesn't fit and we end up with a scrollbar.

Reply:
![image](https://user-images.githubusercontent.com/41635752/188965910-d5976c86-3a45-4ad5-86bd-6d4dc2b5c45b.png)

Reply:
Ah, I see. I'm curious if we even need the tooltip - seems kind of obvious what the "x" does 😆

Reply:
Yeah, probably not really necessary, but it was already there 😄